source-stripe-native/source_stripe_native/priority_capture.py

```diff
@@ -139,16 +145,33 @@ def get_account_priority(self, account_id: str) -> WorkItemPriority:
             "after state reconciliation. Check _reconcile_connector_state implementation and usage."
         )
 
-    def select_prioritized_accounts(self, all_account_ids: List[str], max_worker_count: int) -> set[str]:
+    def select_prioritized_accounts(self, all_account_ids: List[str], max_subtask_count: int) -> set[str]:
         account_priorities = [
             self.get_account_priority(account_id)
             for account_id in all_account_ids
         ]
 
         # Use heapq for efficient partial sorting - only sort as much as needed.
-        top_priority_accounts = heapq.nsmallest(max_worker_count, account_priorities)
+        top_priority_accounts = heapq.nsmallest(max_subtask_count, account_priorities)
+
+        subtask_count = 0
+        selected_accounts: list[WorkItemPriority] = []
 
-        return set([priority.account_id for priority in top_priority_accounts])
+        for account in top_priority_accounts:
+            match account.level:
+                case PriorityLevel.BACKFILL:
+                    subtask_count += 2
+                case PriorityLevel.INCREMENTAL:
+                    subtask_count += 1
+                case _:
+                    raise RuntimeError(f"Unknown priority level {account.level}")
```

Comment:
This case is fine to have, but just curious: Would we ever have an `account.level` that is not `PriorityLevel.BACKFILL` or `PriorityLevel.INCREMENTAL`? Looking at `get_account_priority()` I'd be inclined to say the answer is no, but wanted to confirm.